Source/CarthageKitTests/XcodeSpec.swift

```diff
@@ -176,29 +176,25 @@ class XcodeSpec: QuickSpec {
 		}
 
 		it("should skip projects without shared dynamic framework schems") {
-			let dependency = "SwiftyJSON"
-			let _directoryURL = NSBundle(forClass: self.dynamicType).URLForResource("\(dependency)-2.2.0", withExtension: nil)!
+			let dependency = "SchemeDiscoverySampleForCarthage"
+			let _directoryURL = NSBundle(forClass: self.dynamicType).URLForResource("\(dependency)-0.1", withExtension: nil)!
 			let _buildFolderURL = _directoryURL.URLByAppendingPathComponent(CarthageBinariesFolderPath)
 
-			NSFileManager.defaultManager().removeItemAtURL(_buildFolderURL, error: nil)
+			_ = try? NSFileManager.defaultManager().removeItemAtURL(_buildFolderURL)
 
 			let result = buildInDirectory(_directoryURL, withConfiguration: "Debug", sdkFilter: sdkFilter)
-				|> flatten(.Concat)
-				|> ignoreTaskData
-				|> on(next: { (project, scheme) in
+				.flatten(.Concat)
+				.ignoreTaskData()
+				.on(next: { (project, scheme) in
 					NSLog("Building scheme \"\(scheme)\" in \(project)")
 				})
-				|> wait
+				.wait()
 
 			expect(result.error).to(beNil())
 
-			let macPath = _buildFolderURL.URLByAppendingPathComponent("Mac/\(dependency).framework").path!
 			let iOSPath = _buildFolderURL.URLByAppendingPathComponent("iOS/\(dependency).framework").path!
 
 			var isDirectory: ObjCBool = false
-			expect(NSFileManager.defaultManager().fileExistsAtPath(macPath, isDirectory: &isDirectory)).to(beTruthy())
-			expect(isDirectory).to(beTruthy())
```

Comment:
Because I created [SchemeDiscoverySampleForCarthage](https://github.com/ikesyo/SchemeDiscoverySampleForCarthage) only with iOS framework target. :sweat_smile: I would add Mac framework too.